pyproject.toml

```diff
@@ -66,6 +66,7 @@ mkdocs-git-revision-date-localized-plugin = {version="*", optional=true}
 mkdocs-git-authors-plugin = {version="*", optional=true}
 mkdocs-gitbook = {version="*", optional=true}
 mkdocs-glightbox = {version="*", optional=true}
+xlsxwriter = "^3.1.1"
```

Comment:
Should be moved above optional dependencies. 

Is this dependency needed? We should avoid having too many dependencies, and we already have `openpyxl` as a dependency.

Reply:
re-implemented with openpyxl